svf/lib/SVFIR/SVFType.cpp

```diff
@@ -4,6 +4,11 @@
 namespace SVF
 {
 
+__attribute__((weak))
+u32_t SVFType::getLLVMByteSize() const {
+    return 0;
```

Comment:
Put an assertion to mention implementation and add an abort() method. See SVFValue::toString()